File: 251010/0 나올때까지 출력/print-until-zero-is-given.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int a = 1;
    
    while(a != 0) {
        cin >> a;
        if(a == 0) {
            break;
        }
        cout << a << "\n";
    }
    return 0;
}

File: 251103/0 나올때까지 출력/print-until-zero-is-given.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        // Please write your code here.
        int a;
        while(true) {
            a = int.Parse(Console.ReadLine());
            if(a == 0) {
                break;
            }
            Console.WriteLine(a);
        }
    }
}
